src/utils/logout.ts

```diff
@@ -89,7 +89,7 @@ function redirectToCallbackUrl(
     if (typeof onRedirect === 'function') {
       onRedirect(callbackUrl);
     } else {
-      safeRedirect({ url: callbackUrl, shouldForcePageReload: true });
```

Comment:
why this change?

Reply:
Will make another PR for this

Reply:
https://github.com/multiversx/mx-sdk-dapp/pull/944